### PR TITLE
lint: commentedOutCode false negative on multi-statements

### LINKED ISSUE #668: lint: commentedOutCode false negative on multi-statements
Snippet below has 3 commented-out code lines:
```go
	//rulebase resources
	ruh := rulebases.Handler{Db: datab, WebDir: conf.WebDir, Checker: checker}
	rulebases := api.Group("/rulebases", jwtMiddleware) //base URI
	rulebases.GET("", ruh.GetRulebases)                 //Returns a dictionary with all available Rulebases
	//rulebases.POST("/", postRsHandler)                                //create a rulebase
	//rulebases.DELETE("/:id", deleteRsHandler)                         //delete a rulebase
	//rulebases.PUT("/:setid", putRsHandler)                            //update a rulebase
	rulebases.PUT("/:baseid/documents", ruh.CheckDoc)               //process provided document against rulebase
	rulebases.PUT("/:baseid/documents/:documentid", ruh.CheckDocID) //process document against rulebase
```
They are not detected by the `commentedOutCode` for whatever reason.
They should be reported.

### PR DESCRIPTION
**Pull request by fexolm**
fixes #668

### CODE REVIEW

File: checkers/commentedOutCode_checker.go [hunk 1]
```diff
@@ -77,13 +78,20 @@ func (c *commentedOutCodeChecker) VisitLocalComment(cg *ast.CommentGroup) {
...
+	stmt = strparse.Stmt(fmt.Sprintf("{ %s }", s))
+
 	if stmt == strparse.BadStmt {
 		return // Most likely not a code
 	}
 
-	if !c.isPermittedStmt(stmt) {
-		c.warn(cg)
-	}
+	c.warn(cg)
```
Comment (cristaloleg) [inline checkers/commentedOutCode_checker.go]: Can be pulled into if above.

Comment (fexolm) [reply checkers/commentedOutCode_checker.go]: @cristaloleg fixed

**Comment by quasilyte** [comment]
Anyone tried running it on a bigger code base, like GOROOT?
This checker is very simple to break due to false positives.

**Comment by fexolm** [comment]
@quasilyte no, checking...

**Comment by fexolm** [comment]
Found some false positives. It looks like they were here before. I will add them to a blacklist.

**Comment by fexolm** [comment]
@quasilyte see https://github.com/go-critic/go-critic/pull/840

**Pull request merged**